### ISSUE TITLE
Try to extract common prefix for "+prefix" keymaps.

### ISSUE THREAD

**Issue by a13** [justbur/emacs-which-key#293]
If a prefix keymap contains only commands with a single prefix (say, "projectile-"), optionally show it instead of "+prefix"

**Comment by justbur** [comment]
You can name prefixes with the replacement options.

**Comment by a13** [comment]
> You can name prefixes with the replacement options.

sure, but as I can see, you have to do it manually for every prefix, or you don't?

**Comment by justbur** [comment]
> sure, but as I can see, you have to do it manually for every prefix, or you don't?

yes that's correct. I think it would be more sensible to do this at the time the keys are bound, rather than dynamically through which-key.
